CHANGELOG.md

```diff
@@ -1,10 +1,16 @@
 # Changelog
 
 ## Draft
-
-## 4.5.0 (2020-03-16)
-
-## 4.5.0 (2020-03-16)
+-Added SchemaTranslations.json for cornerstone [#1644|https://github.com/bigcommerce/cornerstone/pull/1644]
+-Facets with & symbol works now [#1654|https://github.com/bigcommerce/cornerstone/pull/1654]
+-Add handlebars class/style variables for account fields [#1655|https://github.com/bigcommerce/cornerstone/pull/1655]
+-Bump lazysizes to 5.2.0 [#1646|https://github.com/bigcommerce/cornerstone/pull/1646]
+-Fix logo in admin bar [#1657|https://github.com/bigcommerce/cornerstone/pull/1657]
+-Special characters to render properly on the cart modal pop up [#1658|https://github.com/bigcommerce/cornerstone/pull/1658]
+-Reset cart quantity to 0 if we get a 404 for the cart [#1661|https://github.com/bigcommerce/cornerstone/pull/1661]
+-Fixed the special characters display issue in the search results page [#1662|https://github.com/bigcommerce/cornerstone/pull/1662]
+-Special characters are not rendered for product review [#1665|https://github.com/bigcommerce/cornerstone/pull/1665]
+-ProductOptions JS for Stencil using KnockOut [#1|https://github.com/bigcommerce/cornerstone/pull/1]
```

Comment:
This seems like an error, this is PR #1 on the repo?

Reply:
<img width="1680" alt="Screenshot 2020-05-11 at 11 48 35" src="https://user-images.githubusercontent.com/50118040/81599576-7f0d6200-937d-11ea-98c0-8b9e5a9a0ef0.png">